homeassistant/components/modbus/const.py

```diff
@@ -58,13 +56,16 @@
 CONF_SWAP_WORD = "word"
 CONF_SWAP_WORD_BYTE = "word_byte"
 CONF_TARGET_TEMP = "target_temp_register"
-CONF_TCP = "tcp"
-CONF_UDP = "udp"
 CONF_VERIFY = "verify"
 CONF_VERIFY_REGISTER = "verify_register"
 CONF_VERIFY_STATE = "verify_state"
 CONF_WRITE_TYPE = "write_type"
 
+CONST_RTUOVERTCP = "rtuovertcp"
```

Comment:
I'd remove the prefix for these. We know they are constants.
```suggestion
RTUOVERTCP = "rtuovertcp"
```

Reply:
Done.